lib/src/util/number.dart

```diff
@@ -86,9 +99,21 @@ num? fuzzyCheckRange(num number, num min, num max) {
 ///
 /// If [number] is [fuzzyEquals] to [min] or [max], it's clamped to the
 /// appropriate value. [name] is used in error reporting.
-num fuzzyAssertRange(num number, int min, int max, [String? name]) {
+double fuzzyAssertRange(double number, int min, int max, [String? name]) {
   var result = fuzzyCheckRange(number, min, max);
   if (result != null) return result;
   throw RangeError.range(
       number, min, max, name, "must be between $min and $max");
 }
+
+/// Return [num1] modulo [num2], using Sass's modulo semantics, which it
+/// inherited from Ruby and which differ from Dart's.
```

Comment:
Should we mention the more general names for each language's semantics here? [Wikipedia says](https://en.wikipedia.org/wiki/Modulo_operation#In_programming_languages) that Ruby/Sass uses floored division, while Dart uses Euclidean division.

Now that I think of it, it doesn't look like we define Sass's modulo semantics in the spec (either the existing ones or the floating point proposal). Should we do that?

Reply:
Good catch! https://github.com/sass/sass/pull/3393